ioctl/newcmd/config/config_set.go

```diff
@@ -5,3 +5,62 @@
 // License 2.0 that can be found in the LICENSE file.
 
 package config
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/iotexproject/iotex-core/ioctl"
+	"github.com/iotexproject/iotex-core/ioctl/config"
+)
+
+var (
+	_configSetUse = map[config.Language]string{
+		config.English: "set VARIABLE VALUE",
+		config.Chinese: "set 变量 值",
+	}
+	_configSetUseCmdShorts = map[config.Language]string{
+		config.English: "Set config fields for ioctl",
+		config.Chinese: "为 ioctl 设置配置字段",
+	}
+	_configSetUseCmdLong = map[config.Language]string{
+		config.English: "Set config fields for ioctl\nValid Variables: [" + strings.Join(_validArgs, ", ") + "]",
+		config.Chinese: "为 ioctl 设置配置字段\n有效变量: [" + strings.Join(_validArgs, ", ") + "]",
+	}
+)
+
+// NewConfigSetCmd is a command to set config fields from iotcl.
+func NewConfigSetCmd(client ioctl.Client) *cobra.Command {
+	use, _ := client.SelectTranslation(_configSetUse)
+	short, _ := client.SelectTranslation(_configSetUseCmdShorts)
+	long, _ := client.SelectTranslation(_configSetUseCmdLong)
+
+	cmd := &cobra.Command{
+		Use:       use,
+		Short:     short,
+		Long:      long,
+		ValidArgs: _validArgs,
+		Args: func(cmd *cobra.Command, args []string) error {
```

Comment:
use cobra.ExactValidArgs.  
add more test datas to test it

Reply:
I'm not sure we can use `cobra.ExactValidArgs` as the user can enter custom args after the config field they are setting. For example `"nsv2height", "44"` which gives the error `invalid argument "44" for "config"` as 44 isn't in the list of valid args.

We could use `cobra.ExactArgs` which checks the number of args?

Thanks

Reply:
yes，`cobra.ExactArgs` can replace check len(args)